zerver/migrations/0482_automatically_follow_unmute_topics_policy_defaults.py

```diff
@@ -0,0 +1,86 @@
+# Generated by Django 4.2.6 on 2023-10-20 23:43
+
+from django.db import migrations, models
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import Max
+
+AUTOMATICALLY_CHANGE_VISIBILITY_POLICY_ON_SEND = 2
+AUTOMATICALLY_CHANGE_VISIBILITY_POLICY_ON_INITIATION = 3
+
+
+def set_default_user_topic_policies(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    RealmUserDefault = apps.get_model("zerver", "RealmUserDefault")
+    UserProfile = apps.get_model("zerver", "UserProfile")
+
+    max_id_realm_user_default = RealmUserDefault.objects.aggregate(Max("id", default=0))["id__max"]
+    max_id_user_profile = UserProfile.objects.aggregate(Max("id", default=0))["id__max"]
+
+    BATCH_SIZE = 5000
```

Comment:
Let's do 1000; there's very little potential benefit to using a value larger than that.

Reply:
Done